packages/react-core/src/components/Divider/Divider.tsx

```diff
@@ -25,13 +25,23 @@ export interface DividerProps extends React.HTMLProps<HTMLElement> {
     xl?: 'insetNone' | 'insetXs' | 'insetSm' | 'insetMd' | 'insetLg' | 'insetXl' | 'inset2xl' | 'inset3xl';
     '2xl'?: 'insetNone' | 'insetXs' | 'insetSm' | 'insetMd' | 'insetLg' | 'insetXl' | 'inset2xl' | 'inset3xl';
   };
+  /** Indicates how the divider will display at various breakpoints. Vertical divider must be in a flex layout. */
```

Comment:
Just a nit and outside the scope, but since we're copying this doc comment, is the bit about it needing to be in a flex layout necessary? I'm assuming so since the vertical divider doesn't really work on its own like the default/horiz variant. If we're going to leave it, technically the parent (or divider itself) just needs a height of some sort, or the parent needs to support `align-items` so that [`align-items: stretch` on the divider](https://github.com/patternfly/patternfly/blob/9da0f628740d4dc7ad78c3a11181d8615196e8ec/src/patternfly/components/Divider/divider.scss#L72) works (this is why it works in flex, but that is supported in grid, too).

Here are a few examples - https://codepen.io/mcoker/pen/ZErzBeg

This is what the [vertical divider core docs](https://www.patternfly.org/v4/components/divider/html#usage) say

> This modifier requires that the parent has an explicit or implicit height, or has a flex or grid based layout parent.

Reply:
@jenny-s51 gotcha! I guess it's a little confusing to me since it can be used outside of a CSS flex layout, but mostly works in the `<Flex>` react layout.